lib/RTCSession.js

```diff
@@ -504,14 +508,17 @@ module.exports = class RTCSession extends EventEmitter
   /**
    * Answer the call.
    */
-  answer(options = {})
+  answer(mediaConnection, options = {})
   {
     debug('answer()');
 
+    if (!isMediaConnection(mediaConnection)) {
+      throw new Exceptions.InvalidMediaConnectionError('Invalid MediaConnection implementation!')
+    }
+
     const request = this._request;
     const extraHeaders = Utils.cloneArray(options.extraHeaders);
     const mediaConstraints = Utils.cloneObject(options.mediaConstraints);
-    const mediaStream = options.mediaStream || null;
```

Comment:
We still want to accept `MediaStream`'s, what we are not doing anymore is `getUserMedia`

Reply:
Custom streams can be set with the new mediaconnection interface